resources/sdk/purecloudgo/templates/apiclient.mustache

```diff
@@ -106,20 +147,8 @@ func (c *APIClient) CallAPI(path string, method string,
 	fileBytes []byte,
 	pathName string) (*APIResponse, error) {
 
-	// Build request URL w/query params
 	urlString := path + "?"
-	if queryParams != nil {
-		for k, v := range queryParams {
-			if v != "" {
-				urlString += fmt.Sprintf("%v=%v&", url.QueryEscape(strings.TrimSpace(k)), url.QueryEscape(strings.TrimSpace(v)))
-			}
-		}
-	}
-	urlString = urlString[:len(urlString)-1]
-	u, err := url.Parse(urlString)
-	if err != nil {
-		return nil, err
-	}
+	u := getConfUrl(c.configuration, pathName, "", c.configuration.Host)
```

Comment:
what would be the value of  pathname here

the logic of urlString := path + "?" 

urlString = urlString[:len(urlString)-1]
	u, err := url.Parse(urlString)
	if err != nil {
		return nil, err
	}
	
	is also ignored in getconfurl method. this might create regression problems .